static/rally/index.html

```diff
@@ -189,5 +189,22 @@
 </script>
 <!-- End Google Analytics -->
 
+<script>
+  var autocomplete;
+
+  function initAutocomplete() {
+    // Create the autocomplete object, restricting the search to geographical
+    // location types.
+    autocomplete = new google.maps.places.Autocomplete(
+        /** @type {!HTMLInputElement} */(document.getElementById('address_autocomplete')),
```

Comment:
this is unusual syntax. there's an inline type declaration? 

Also is there a reason for the extra set of parens around `getElementbyId`?